src/NextGen/DonationForm/resources/propTypes.ts

```diff
@@ -1,21 +1,117 @@
-import {Element, Field, Group, SelectOption} from '@givewp/forms/types';
-import {UseFormRegisterReturn} from 'react-hook-form';
-import {FC} from 'react';
+import {Element, Field, FormDesign, Gateway, Group, Section as SectionType, SelectOption} from '@givewp/forms/types';
+import {FieldErrors, UseFormRegisterReturn} from 'react-hook-form';
+import {FC, FormHTMLAttributes, ReactNode} from 'react';
 
 export interface FieldProps extends Field {
     inputProps: UseFormRegisterReturn;
-    Label: FC;
-    ErrorMessage: FC;
+    Label: FC<FieldLabelProps | {}>;
+    ErrorMessage: FC<FieldErrors | {}>;
```

Comment:
Why the optional empty object? Same for the `NameProps` below.

Reply:
This lets us render the components (when props are already applied) in our templates, yet still have the props available:

<img width="467" alt="Screen Shot 2022-11-29 at 10 55 34 PM" src="https://user-images.githubusercontent.com/10138447/204704144-9548ef0c-fcfa-4328-8f4f-d0d433ed9701.png">